extension/index.googleFunction.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * If you are deploying extension module as a serverless function,
+ * this will be the main javascript file.
+ *
+ * This function has tested as Google Cloud Function
+ *
+ * Entry point: extensionTrigger
+ */
+
+const paymentController = require('./src/api/payment/payment.controller')
```

Comment:
This code does not look correct to me, according to overriding the existing function of the httpUtils. 
we did similar work with Jude for notification endpoint.
cc: @JudeNiroshan 

You might check the existing lambda function.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/528/commits/336ab887912e7893b6d9938461d995ee448830af